blog/2023-10-29-ratify-1.0.0.mdx

```diff
@@ -0,0 +1,80 @@
+---
+slug: ratify-v1.0.0-is-available
+title: Ratify v1.0.0 is available! An extensible verification framework for software supply chain security
+authors: [ratify]
+tags: [verification]
+---
+
+The Ratify maintainers are proud to announce the first major release v1.0.0 for production use!
+
+Ratify is an extensible verification framework for container images and other artifacts that can examine and use custom policies that you create to approve deployments in Kubernetes. Ratify can use and coordinate any number of custom verifiers for image metadata like signatures, SBOMs, vulnerability scan reports, and so on.
+It helps users to validate the image integrity and authenticity in containers secure supply chain, ensures your deployments are built from a trusted entity and that images haven't been tampered with since their creation.
+![](/img/ratify-e2e.png)
+
+Ratify was [launched and open-sourced](https://cloudblogs.microsoft.com/opensource/2021/12/09/ratify-container-supply-chain-in-kubernetes/) on December 9, 2021. After the fast iteration with 18 releases over the past two years, the core framework becomes mature and extensible now and reaches enterprise-grade stability. Users can easily run Ratify as an External Data Provider for Gatekeeper admission controller on Kubernetes with Helm.
+
+## Notable capabilities in this Release
+
+- Ratify as a CLI binary for verifying artifacts stored in a registry
+- Verifier, Store, Certificate Store, and Policy CRDs for simple Ratify configuration
+- TLS certificate management and rotation for mTLS service-to-service communication
+- Out-of-the-box observability: logging and prometheus metrics support
+- Extensible plugin design to support new verifier and referrer store plugins
+- High availability support (Experimental) 
+    
```

Comment:
wonder if we need to mention the verifiers Ratify already supports?

Reply:
Updated and added verifiers